packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -439,15 +621,16 @@ export class ModularChangeFamily
 			change1?.nodeExistsConstraint ?? change2?.nodeExistsConstraint;
```

Comment:
Could this logic be problematic if they both have a constrsaint but the constraint is only violated in one of the changesets?

Reply:
Also:
```suggestion
			change1.nodeExistsConstraint ?? change2.nodeExistsConstraint;
```

Reply:
Currently we drop all tree changes from a conflicted changeset when composing, so we can assume that neither are violated.

Reply:
Ah, right, we get to do that because of the ways we're currently using compose:
- Either we don't need to preserve changes with violated constraints (when rebasing over compositions or when producing a delta for a composition)
- Or we need to preserve changes but we can tell that the constraints are for sure not violated (when compositing a local transaction)